site/content/posts/2021-06-7-Windows-Support.md

```diff
@@ -0,0 +1,102 @@
+
+Since the Windows support is new, you need to either have the latest version of Sonobuoy or specify the image via a flag. For explicitness, I’ll show the flag but realize that it is optional if you’ve updated your CLI client.
+
+```
+url=https://raw.githubusercontent.com/vmware-tanzu/sonobuoy-plugins/master/examples/windows-plugin/plugin.yaml
+img=sonobuoy/sonobuoy:v0.51.0
+
+sonobuoy run --plugin $url --sonobuoy-image=$img --wait
+```
+
+Once that command completes you can check the status and/or download results:
+
+```
+sonobuoy status
+tarball=$(sonobuoy retrieve)
+```
+
+Once you have the tarball of results you can either extract the contents yourself or just inspect it with `sonobuoy results`. Check out our previous [post](https://sonobuoy.io/simplified-results-reporting-with-sonobuoy/) about different ways to inspect the results. The plugin reported the actual events output from the Windows machine:
+
+```
+Sonobuoy results $tarball --mode=detailed
+
+
+
+   ProviderName: Microsoft-Windows-DHCPv6-Client
+
+TimeCreated                     Id LevelDisplayName Message                    
+-----------                     -- ---------------- -------                    
+5/25/2021 1:05:39 AM          1008 Error            An error occurred in ini...
+5/25/2021 1:05:39 AM          1008 Error            An error occurred in ini...
+
+
+   ProviderName: Microsoft-Windows-Security-Auditing
+
+TimeCreated                     Id LevelDisplayName Message                    
+-----------                     -- ---------------- -------                    
+5/25/2021 1:05:39 AM          4688 Information      A new process has been c...
+
+
+   ProviderName: Microsoft-Windows-PowerShell
+…
+…
+…
+```
+
+That’s all it takes to run a Windows plugin: that is to say, nothing extra at all! Now that Sonobuoy publishes the Windows images as part of a multi-arch image, each node will automatically pull the correct version (Linux/Windows). The only thing you need is a plugin that targets a Windows compatible image and a cluster to run it on.
+
+## How To Help
+
+One of the most easiest ways you can help contribute is to simply tell us what you want, for instance:
+What test frameworks or output formats are common in your Windows workflows?
+What kind of logs or system information do you routinely find yourself needing from the Windows machines?
+What part of Kubernetes do you routinely/specifically want to test? All of Conformance to see where Windows isnt compatible? Just Windows features? Some other subset?
+
+If telling us these things is good, helping us actually build or test them is even better. You can put up an issue or pull request to Sonobuoy or sonobuoy-plugins repos with complete code or even just ideas and we can help build tools to help the community together.
```

Comment:
Please link to the repos for easy access.
"Put up an issue" should be "create an issue"